backend/users/client.py

```diff
@@ -16,6 +16,14 @@
 }
 """
 
+GET_USERS_BY_NAME = """
```

Comment:
I would name it more specific to what it does so 🤔 

```suggestion
SEARCH_USERS = """
```